build-system/tasks/runtime-test.js

```diff
@@ -334,20 +326,29 @@ gulp.task('test', 'Runs tests',
     'grep': '  Runs tests that match the pattern',
     'files': '  Runs tests for specific files',
     'randomize': '  Runs entire test suite in random order',
-    'testlist': '  Runs tests specified in JSON by supplied file',
```

Comment:
Remove this from `argvMessages` starting in line 121.

Reply:
done